README.md

```diff
@@ -123,6 +123,15 @@ __add_custom_layers__: bool
 	  Parameters for these custom layers should be filled manually by editing the mlmodel  
 	  or the 'custom_conversion_functions' argument can be used to do the same during the process of conversion
 
+__custom_conversion_fuctions__: dict (str: function)  
+	  Specify custom function to be used for conversion for given op.
+        User can override existing conversion function and provide their own custom implementation to convert certain ops.
```

Comment:
can you explain a bit more about what values can the `key` take? for instance it could either be the type of an onnx op, in which case the converter will call the user function whenever an op of that type is encountered. 
Or it could also be the unique name of an op, in which case the function will be called just for that particular op.